README.md

```diff
@@ -1,10 +1,11 @@
 # Hackerrank
 Find the solved codes of HackerRank here by navigation in the particular Section.
+Made as part of thirty days of code.
```

Comment:
@surya1701 This repo is not particularly for 30 days of code. It can have solutions form any section of hackeRank.
Please modify or remove this line .